skiko/src/commonMain/kotlin/org/jetbrains/skiko/SkiaLayer.kt

```diff
@@ -9,25 +9,13 @@ expect open class SkiaLayer {
     var fullscreen: Boolean
     var transparency: Boolean
 
-    var renderer: SkiaRenderer?
+    var app: SkikoApp?
 
     fun needRedraw()
 }
 
-interface SkiaRenderer {
-    fun onRender(canvas: Canvas, width: Int, height: Int, nanoTime: Long)
-}
-
-open class GenericRenderer(
-    val layer: SkiaLayer,
-    val app: SkiaRenderer
-): SkiaRenderer {
-    override fun onRender(canvas: Canvas, width: Int, height: Int, nanoTime: Long) {
-        val contentScale = layer.contentScale
-        canvas.scale(contentScale, contentScale)
-        app.onRender(canvas, (width / contentScale).toInt(), (height / contentScale).toInt(), nanoTime)
-        layer.needRedraw()
-    }
+fun SkiaLayer.setApp(app: SkikoApp) {
```

Comment:
SkiaLayer usually used per window.

And App usually means the whole application, not only the content of some particular Window.

Maybe we rename SkikoApp to something else? SkikoScene or SkikoContent for example.